src/wallet/wallet.cpp

```diff
@@ -3293,6 +3296,12 @@ std::string CWallet::GetWalletHelpString(bool showDebug)
 
 bool CWallet::InitLoadWallet()
 {
+    if (GetBoolArg("-disablewallet", false)) {
```

Comment:
Some constants are too obvious to have a default value. In the future this might change for `-disablewallet` and we should introduce the default value.